ast/policy.go

```diff
@@ -261,28 +261,22 @@ func (mod *Module) Copy() *Module {
 	cpy := *mod
 	cpy.Rules = make([]*Rule, len(mod.Rules))
 
-	nodes := make(map[Node]Node)
+	nodes := make(map[Node]Node, len(mod.Rules)+len(mod.Imports)+1 /* package */)
```

Comment:
The three `if nodes != nil` below hadn't been necessary without this preallocation; but since we know the size, we might as well tell `make`.